src/pipeline.cpp

```diff
@@ -25,6 +25,8 @@
 #define MODEL_INPUT_X 224
 #define MODEL_INPUT_Y 224
 
+using namespace std;
```

Comment:
Could we take this out?

Reply:
no problem, that was used for testing but has been removed.